Jenkinsfile

```diff
@@ -9,67 +9,86 @@ pipeline {
-                                curl -X POST -H 'Content-type: application/json' --data-binary "@minikube_data.txt"  https://hooks.slack.com/services/$SLACK_TOKEN
-                                sh "exit 1"
-                            fi
+                        kubectl get pods -o=custom-columns=NAME:.metadata.name -A | grep assisted-service | xargs -I {} sh -c "kubectl logs {} -n  assisted-installer > test_dd.log"
+                        mv test_dd.log $WORKSPACE/assisted-service.log || true
                         '''
-            }
-        }
 
-        stage('clear deployment') {
-            steps {
-                sh 'docker image prune -a -f'
-                sh 'make clear-deployment'
-            }
-        }
+                        echo 'Get postgres log'
+                        sh '''kubectl  get pods -o=custom-columns=NAME:.metadata.name -A | grep postgres | xargs -I {} sh -c "kubectl logs {} -n  assisted-installer > test_dd.log"
+                        mv test_dd.log $WORKSPACE/postgres.log || true
+                        '''
 
-        stage('Build') {
-            steps {
-                sh '''export PATH=$PATH:/usr/local/go/bin; make build-image build-assisted-iso-generator-image'''
-            }
-        }
+                        echo 'Get scality log'
+                        sh '''kubectl  get pods -o=custom-columns=NAME:.metadata.name -A | grep scality | xargs -I {} sh -c "kubectl logs {} -n  assisted-installer > test_dd.log"
+                        mv test_dd.log $WORKSPACE/scality.log || true
+                        '''
 
-        stage('Deploy for subsystem') {
-            steps {
-                withCredentials([usernamePassword(credentialsId: 'dockerio_cred', passwordVariable: 'PASS', usernameVariable: 'USER')]) {
-                    sh '''docker login docker.io -u $USER -p $PASS'''
+                        echo 'Get createimage log'
+                        sh '''kubectl  get pods -o=custom-columns=NAME:.metadata.name -A | grep createimage | xargs -I {} sh -c "kubectl logs {} -n  assisted-installer > test_dd.log"
+                        mv test_dd.log $WORKSPACE/createimage.log || true
+                        '''
+                        clear_slave()
+                    }
                 }
-                sh '''export PATH=$PATH:/usr/local/go/bin; make jenkins-deploy-for-subsystem'''
-                sh '''# Dump pod statuses;kubectl get pods -A'''
-            }
-        }
-
-        stage('Subsystem-test') {
-            steps {
-                sh '''export PATH=$PATH:/usr/local/go/bin;make subsystem-run'''
-            }
-        }
+                stage('Test onprem') {
+                    agent { label 'centos_worker' }
+                    steps {
+                        check_if_minikube_is_running()
+
+                        clear_slave()
+
+                        sh '''export PATH=$PATH:/usr/local/go/bin; make build-onprem'''
```

Comment:
We saw that this part require to copy things from the regular build no?

Reply:
apparently no, the copy was from the wrong location due to Raz's bug. 
ill

Reply:
The bug is not related, i'm talking about https://github.com/openshift/assisted-service/blob/master/Dockerfile.assisted-service-onprem#L9-L10